[2021-08-28 00:09:25] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 00:43:25] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/keyworks/app/storage/views/23f7c75a680f7512da0c89242ef7b99b:69 Stack trace: #0 /var/www/keyworks/app/storage/views/23f7c75a680f7512da0c89242ef7b99b(69): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/keywor...', 69, Array) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/keywor...') #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/keywor...', Array) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/keywor...', Array) #4 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/keyworks/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(137): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'noticiaInterna', Object(Illuminate\View\View)) #10 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'noticiaInterna', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('manifestacao-do...') #12 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2021-08-28 01:02:11] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 01:02:12] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 01:02:13] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 01:02:14] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 01:02:14] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 01:02:15] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 01:02:16] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 01:02:17] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 01:16:50] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 01:21:56] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 01:33:13] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 01:42:55] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in /var/www/keyworks/app/storage/views/23f7c75a680f7512da0c89242ef7b99b:69 Stack trace: #0 /var/www/keyworks/app/storage/views/23f7c75a680f7512da0c89242ef7b99b(69): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', '/var/www/keywor...', 69, Array) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/var/www/keywor...') #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/var/www/keywor...', Array) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/var/www/keywor...', Array) #4 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents() #5 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render() #6 /var/www/keyworks/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(137): Illuminate\Http\Response->setContent(Object(Illuminate\View\View)) #7 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View)) #8 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request)) #9 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'noticiaInterna', Object(Illuminate\View\View)) #10 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'noticiaInterna', Array) #11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('noticia') #12 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array) #13 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable() #14 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request)) #15 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #16 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #17 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #18 {main} [] []
[2021-08-28 03:19:45] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 03:19:45] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 04:33:55] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1439 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 04:34:35] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 04:40:45] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 04:48:12] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 06:21:47] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 06:21:48] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 06:21:48] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 06:41:38] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 06:41:38] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 06:41:39] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 06:41:40] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 06:41:41] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 06:41:42] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 06:41:43] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 06:41:45] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 06:41:46] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 06:41:47] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 06:41:48] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 06:41:49] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 06:41:49] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 06:41:51] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 06:41:51] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 06:41:52] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 06:41:53] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 06:41:54] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 07:03:25] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 07:03:38] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 07:25:35] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 07:25:36] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 07:25:36] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 07:25:37] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 07:25:37] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 07:29:13] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 07:29:23] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 07:29:23] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 08:30:04] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1439 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 09:09:27] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 09:22:41] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 09:22:43] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1439 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 09:52:25] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 11:49:40] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 12:27:15] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 12:31:02] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 12:39:40] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 12:57:34] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 13:31:16] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 13:34:49] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 13:34:53] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1439 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 13:34:59] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 13:44:10] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 15:00:21] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 16:55:41] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 17:04:57] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 17:04:57] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 19:31:51] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1439 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 19:55:36] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 20:16:50] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 20:17:04] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 20:20:34] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 20:58:38] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 20:58:49] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 21:09:37] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1439 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 21:59:02] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 22:33:56] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 22:39:33] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1439 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 22:39:37] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1439 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 22:39:40] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1439 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 22:39:43] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1439 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 22:39:47] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1439 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 22:39:50] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1439 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 22:39:53] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1439 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 22:39:56] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1439 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 22:40:00] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1439 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 22:40:03] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1439 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 22:40:06] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1439 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 22:40:10] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1439 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 22:40:13] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1439 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 22:40:16] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1439 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 22:40:20] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1439 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 22:40:23] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1439 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 22:40:26] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1439 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 22:44:22] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 22:46:24] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 23:02:43] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1439 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\MethodNotAllowedException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 23:15:22] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
[2021-08-28 23:19:01] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429 Stack trace: #0 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException)) #1 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request)) #2 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request)) #3 /var/www/keyworks/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request)) #4 /var/www/keyworks/index.php(49): Illuminate\Foundation\Application->run() #5 {main} [] []
